src/views/docs-view/docs-view.module.css

```diff
@@ -0,0 +1,3 @@
+.mdxContent {
+	margin-top: 48px;
```

Comment:
Long-winded note on spacing in CSS:

> TLDR I'm trying to stick to `margin-top`. And avoid components defining the spacing around themselves, [per Notion](https://www.notion.so/hashicorp/CSS-Modules-753f2647f7aa4a598e28990d9f0d05a8).

---


I had trouble reasoning around spacing, as we had a combination of padding and margin across various `top` and `bottom` properties. Our [Notion guidelines](https://www.notion.so/hashicorp/CSS-Modules-753f2647f7aa4a598e28990d9f0d05a8) mention `No space around components`, so I started with that. I think a decent additional guidelines would be to consistently apply spacing between elements with EITHER `margin/padding-top` or `margin/padding-bottom` - having a mix of the two can make it difficult to reason about spacing, as we have to think about combining pairs of elements. I've gone with `margin-top` on the second element, as I find it tends to be more predictable with respect to optional elements and spacing at the end of a section.

Happy to dive further into spacing issues, this is definitely an area I would love to get involved in in terms of finding consistent practices, which I think would be best to establish across design and engineering so that we have a shared mental model. For now I'm not trying to dive too deep so as to keep the scope of this PR reasonable.

Reply:
I generally agree re spacing considerations, but have preferred `margin-bottom` for no other reason than preference 😄 Thanks for the insight here!